java/sage/FSManager.java

```diff
@@ -478,7 +492,12 @@ public void releaseISOMount(java.io.File mountDir)
         }
       }
       else
-        IOUtils.exec2(new String[] { "umount", mountDir.getAbsolutePath() }, false);
+      {
+        if (Sage.LINUX_IS_ROOT)
+          IOUtils.exec2(new String[]{"umount", mountDir.getAbsolutePath()}, false);
+        else
+          IOUtils.exec2(new String[]{"sudo","umount", mountDir.getAbsolutePath()}, false);
```

Comment:
This will hang indefinitely waiting for a password if the sudoers configuration is not set to not require a password for at least this command.